docker/ubuntu1604/docker-entrypoint.sh

```diff
@@ -17,10 +17,5 @@ then
   [ -d /home/jenkins/.ssh ] && chown ${UID} /home/jenkins/.ssh
 fi
 
-# update alba & arakoon packages to latest/greatest
```

Comment:
keep this the same as for 14.04 (couldn't do this at the time this Dockerfile was created)